src/utils/FileUtils.ts

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2021 Šimon Brandner <simon.bra.ag@gmail.com>
```

Comment:
copyright of the foundation would be maintained for the copy/paste bits, in addition to your copyright.